ports/stm32/eth.c

```diff
@@ -380,6 +430,16 @@ STATIC int eth_mac_init(eth_t *self) {
                 | (1 << RX_DESCR_3_IOC_Pos) // Interrupt Enabled on Completion
         ;
         eth_dma.rx_descr[i].rdes0 = (uint32_t)&eth_dma.rx_buf[i * RX_BUF_SIZE]; // buf 1 address
+        #elif defined(STM32H5)
+        // TODO: Check STM32H5
+        // eth_dma.rx_descr[i].rdes1 = 0;  // reserved
+        // eth_dma.rx_descr[i].rdes2 = 0;
```

Comment:
Can this TODO and the commented-out code be removed?

Reply:
Done